core/src/main/java/software/amazon/awssdk/core/runtime/transform/StandardErrorUnmarshaller.java

```diff
@@ -100,40 +101,40 @@ public String getErrorPropertyPath(String property) {
         return "ErrorResponse/Error/" + property;
     }
 
-    public AmazonServiceException standardErrorPathException(String errorCode, Node in, XPath xpath) throws Exception {
+    public SdkServiceException standardErrorPathException(String errorCode, Node in, XPath xpath) throws Exception {
 
         String errorType = asString("ErrorResponse/Error/Type", in, xpath);
         String requestId = asString("ErrorResponse/RequestId", in, xpath);
         String message = asString("ErrorResponse/Error/Message", in, xpath);
 
-        AmazonServiceException ase = newException(message);
-        ase.setErrorCode(errorCode);
-        ase.setRequestId(requestId);
+        SdkServiceException exception = newException(message);
+        exception.errorCode(errorCode);
+        exception.requestId(requestId);
 
-        if (errorType == null) {
-            ase.setErrorType(AmazonServiceException.ErrorType.Unknown);
-        } else if (errorType.equalsIgnoreCase("Receiver")) {
-            ase.setErrorType(AmazonServiceException.ErrorType.Service);
-        } else if (errorType.equalsIgnoreCase("Sender")) {
-            ase.setErrorType(AmazonServiceException.ErrorType.Client);
+        if ("Receiver".equals(errorType)) {
```

Comment:
where do the values "Receiver" and "Sender"? Also document it

Reply:
I'll add documentation.